MonoGame.Framework.Content.Pipeline/Processors/ModelProcessor.cs

```diff
@@ -99,9 +99,9 @@ public override ModelContent Process(NodeContent input, ContentProcessorContext
             // Perform the processor transforms.
             if (RotationX != 0.0f || RotationY != 0.0f || RotationZ != 0.0f || Scale != 1.0f)
             {
-                var rotX = Matrix.CreateRotationX(MathHelper.ToRadians(RotationX));
-                var rotY = Matrix.CreateRotationY(MathHelper.ToRadians(RotationY));
-                var rotZ = Matrix.CreateRotationZ(MathHelper.ToRadians(RotationZ));
+                var rotX = Matrix.CreateRotationX((float)MathHelperD.ToRadians(RotationX));
```

Comment:
Model processor changes should be in another PR. Also, I don't quite get the relevance of casting a double to float here while the original MathHelper does the same thing internally?